search-service/src/main/kotlin/com/egm/stellio/search/csr/web/ContextSourceRegistrationHandler.kt

```diff
@@ -59,7 +60,7 @@ class ContextSourceRegistrationHandler(
      * Implements 6.8.3.1 - Create ContextSourceRegistration
      */
     @PostMapping(consumes = [MediaType.APPLICATION_JSON_VALUE, JSON_LD_CONTENT_TYPE])
-    suspend fun create(
+    suspend fun createContextSourceRegistration(
```

Comment:
I disagree with this move. In a CSRHandler, we know that create / query / ... applies to CSR.